### ISSUE TITLE
Certain Dialog Boxes Missing Some Labels for Screen Readers

### ISSUE THREAD

**Issue by trodick** [trypsynth/fedra#60]
I'm running Fedra from the latest source, commit 3CA561E, and I've noticed some regressions in terms of labels on the Content Warning checkbox (New Post dialog) as well as the radio buttons used when voting on polls. Running NVDA 2025.3.3.54605, Windows 11 Pro 25H2 (OS build 26200.8037). The Content Warning checkbox now just says "Check checkbox," and the poll-voting radio buttons announce themselves as "Radio button radio button." Haven't tested with JAWS or Narrator, but I can if necessary. Thanks!

**Comment by trypsynth** [comment]
That's a wxWidgets bug, clone wxWidgets locally including all its submodules, set the WXWIDGETS_DIR env var to where it lives, and try building again.
